src/test/datascience/jupyter/kernels/installationPrompts.vscode.test.ts

```diff
@@ -322,7 +322,15 @@ suite('DataScience Install IPyKernel (slow) (install)', function () {
         await verifyIPyKernelPromptDisplayed(prompt, venvPythonPath.fsPath);
         await verifyErrorInCellOutput(notebookDocument, venvPythonPath.fsPath);
 
-        await sleep(1_000);
+        // Wait for the prompt to display
+        // await waitForCondition(
+        // () => {
+        // return prompt.displayed;
+        // },
+        // delayForUITest,
+        // 'Failed to display prompt'
+        // );
+        await sleep(3_000);
```

Comment:
weird, probably something changed in vscode causing something to slow down.
as i don't see any changes in our code for this test to stop working consistently within a day.

Reply:
Yeah it's odd. At first I added some logging, upped the sleep a bit, and got a totally green run. But then I was playing with it some more and now the errors look like they are back? Will see...